bio/bcftools/sort/wrapper.py

```diff
@@ -0,0 +1,54 @@
+__author__ = "Filipe G. Vieira"
+__copyright__ = "Copyright 2021, Filipe G. Vieira"
```

Comment:
Sorry, 2020 is not yet over...
```suggestion
__copyright__ = "Copyright 2020, Filipe G. Vieira"
```